galley/pkg/config/processor/metadata/metadata.yaml

```diff
@@ -519,6 +519,8 @@ snapshots:
       - "istio/security/v1beta1/authorizationpolicies"
       - "k8s/core/v1/namespaces"
       - "k8s/core/v1/services"
+      # TODO: this is for client analysis, and not needed for Galley. We should add support for a separate collection for this
```

Comment:
If we're adding a TODO comment, we should link it to a github issue.

Reply:
Agreed. Not sure I have quite enough context to properly describe the need to decouple the two collections. If you or @ozevren could create that issue, that's be best.

Is this something we might tackle soon?

Reply:
https://github.com/istio/istio/issues/17227